articles/architecture-scenarios/implementation/b2c/branding.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Branding
+description: Understand how you can configure Auth0 items to reflect your brand and desired user experience
+toc: true
+topics:
+    - b2c
+    - ciam
+    - branding
+    - universal-login
+    - login-pages
+    - change-password-pages
+    - custom-domains
+    - error-pages
+contentType: concept
+useCase:
+    - user-logout
+---
+# Branding
+
+You can customize Auth0 with a look and feel that aligns with your organization's brand requirements and user expectations. The consistency in appearance improves your customer's experience, giving them peace of mind that the pages they're using can be trusted and are secure.
+
+Some of the items that you can brand include:
+
+* Login, Change Password, and Error pages
+* Custom domains
+* Email templates
+
+Auth0 also supports [internationalization (I18N)](/i18n) and localization (L10N), both of which are important if you work with an international clientele.
+
+## Design considerations
+
+For some users, the out-of-box experience -- perhaps with just a touch of alteration -- is more than sufficient. However, for those wanting more control over appearance and user experience, Auth0 provides extensive flexibility when it comes to customizing and configuring the user experience look and feel you need.
+
+When considering the items you want to brand, as well as how best to brand these items, you'll want to consider the following.
+
+* Will you brand your login page?
+* Will you need to localize your login page?
+* Will you customize emails so that they're not just branded, but vary based on user preference?
+* How will users know that they're still on your domain when they see your login page?
+* What do you need to do to provide additional browser security (e.g., implement Extended Validation)?
+* How will you direct users in the event of errors?
+
+The answers to these questions and more are detailed in the [Universal login and login pages](#universal-login-and-login-pages), [Custom domains](#custom-domains), and [Email templates](#email-templates) sections below. 
```

Comment:
Again, the reader is left to work things out for themselves, whereas the original document provided prescribed and directed guidance (https://docs.google.com/document/d/1wHEmRsPVciJJVuyssBH3RyO1mLYIGqLj34-vhVFEODU/edit#bookmark=id.j5qsnx3vzh6o). If we can re-instate the original text I think that would be best 👍

Reply:
Done